app/controllers/review_controller.rb

```diff
@@ -64,6 +64,21 @@ def submit_for_2i
     end
   end
 
+  def revert_to_draft
+    status = "draft"
+    previous_status = @step_by_step_page.status
+
+    if @step_by_step_page.update(
+      reviewer_id: nil,
+      review_requester_id: nil,
+      status: status,
+    )
+      generate_change_note("Reverted from #{previous_status} to draft")
+
+      redirect_to step_by_step_page_path(@step_by_step_page.id), notice: "Step by step page was successfully #{status.humanize.downcase}."
```

Comment:
This will render `Step by step page was successfully draft`.  Is it possible to change this text, maybe to `Step by step page was successfully reverted to draft`?

Reply:
👍